test/test_linalg.py

```diff
@@ -3942,8 +3942,8 @@ def test_renorm_ps(self, device):
             expected = x / x.norm(p, 0, keepdim=True).clamp(min=1)
             self.assertEqual(res, expected, msg="renorm failed for {}-norm".format(p))
 
-    @onlyCPU
     @skipCPUIfNoLapack
+    @skipCUDAIfNoMagmaAndNoCusolver
     @dtypes(torch.float32, torch.float64, torch.complex64, torch.complex128)
     def test_orgqr(self, device, dtype):
```

Comment:
How good is this test? Do we need to revise its coverage while we're working on it?

Reply:
I think this test is as good as it could be because I wrote it 😀
https://github.com/pytorch/pytorch/pull/50502/files#r556989761

It covers 0x0 matrices, 0 batch dimension, normal single matrices, and batched ones.